website/src/content/deployment-examples/google-cloud-storage.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Deploying to Google cloud
+---
+
+This deployment example refers to [Google cloud Static Web Apps](https://codelabs.developers.google.com/codelabs/cloud-webapp-hosting-gcs#0)
+
+# Prerequisites
+
+Before you get started, you need to have:
+
+- A [google](https://accounts.google.com/SignUp) account.
+- A custom domain
+- A [commercetools](https://docs.commercetools.com/getting-started) account and a Project.
+- A Custom Application [configured in the Merchant Center](https://docs.commercetools.com/merchant-center/managing-custom-applications).
+
+# Configuration
+
+In your Custom Application config, provide the [Custom Application ID](/api-reference/application-config#envproductionapplicationid) you got when you configured the Custom Application in the Merchant Center.
+
+
+```json title="custom-application-config.json"
+{
+  "env": {
+    "production": {
+      "applicationId": "ckvtahxl90097sys6har1e6n3",
+      "url": "https://<custom_domain>"
+    }
+  }
+}
+```
+
+## Using environment variables
+
+In case you want to avoid hardcoding certain values, for example the Application ID, or the Project key, you can [use variable placeholders](/api-reference/application-config#using-variable-placeholders) in your Custom Application config.
+
+```json title="custom-application-config.json" highlightLines="4"
+{
+  "env": {
+    "production": {
+      "applicationId": "${env:APPLICATION_ID}",
+      "url": "https://<custom_domain>"
+    }
+  }
+}
+```
+
+# Set up Google Cloud
+
+To host the production bundles of your Custom Application you need to create a [Cloud Storage bucket](https://console.cloud.google.com/storage) in Google cloud and [configure](https://cloud.google.com/storage/docs/hosting-static-website#create-bucket) the bucket.
+
+Make sure you are the owner and administrator of your domain.
+
+<Info>
+
+Merchant Center only allows accessing your Custom Application using a secure protocol (HTTPS) so your domain must support it with a valid [SSL certificate](https://support.google.com/domains/answer/7630973?hl=en&visit_id=637951191082633113-3999318342&rd=1).
+
+</Info>
+
+
+# Connect Google cloud with GitHub Actions
+
+One of the ways to deploy to google cloud is to use [Cloud Build](https://cloud.google.com/build/docs/automating-builds/create-manage-triggers#connect_repo) deployment service. This service enables GitHub integration.
+
+<Info>
+
+Make sure to also install the [Google Cloud GitHub App](https://github.com/apps/google-cloud-build) to grant access to your repository.
+
+</Info>
```

Comment:
I'm not sure we should recommend using Cloud Build, as this is usually for building docker images etc.

For static files, you just need to upload them to the bucket. Maybe we should recommend using a github action: https://github.com/google-github-actions/upload-cloud-storage

So the github integration could be about setting up the github action to upload the static files to the bucket.

Reply:
Thank you, all updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2740/commits/8dd0ba6dfdccc43d45295f7b963abbef68ebf649